demos/polls/README.rst

```diff
@@ -7,20 +7,57 @@ similar to django one.
 Installation
 ============
 
-Install the app::
+Prepare development environment
+-------------------------------
+
+We suggest you to start by creating an isolated Python working environment::
+
+    $ python3 -m venv env
+    $ source env/bin/activate
+
+
+Install the app and it's requirements::
 
     $ cd demos/polls
     $ pip install -e .
 
-Create database for your project::
 
-    bash sql/install.sh
+Prepare database
+----------------
+Install Postgresql database server: http://www.postgresql.org/download/
+(to use postgres in more isolated way you may also use docker. We will explain how to do that
+after environment variables setup.)
+
+Create ``.env`` file in ``polls`` folder with db related configuration values. It is a common practice to not
```

Comment:
It's better to commit `.env.example` and then instruct user to copy it and then edit. Oh and commit `.gitignore` file next to it.